Yoast/YoastSEO.js
# Add tree visualization to the devtool

## Summary

This PR can be summarized in the following changelog entry:

* [non-user-facing] Add visualization of the tree.

## Relevant technical choices:

* Creating an option to build the tree every text change.
* Represent it using react component: `react-json-view`.

## Screenshots
<img width="986" alt="jsontree" src="https://user-images.githubusercontent.com/35524806/52556815-e7256f00-2ded-11e9-9e2c-bbc17c408e33.png">

## Test instructions

This PR can be tested by following these steps:

* Go to the path `cd examples/webpack`.
* Install dependencies `yarn`.
* Start the dev server `yarn start`.
* Test the option `Build the tree`. While on it should build the tree every text change. While off it should do nothing. Turning it back on should (most of the time, there can be a timing thing with the worker results) re-build the tree.
* Check the tree representation itself.

Fixes #2122
